### PR TITLE
Hl 1269 versionseriesid

### PR DESCRIPTION
**Pull request by rikuke**
## Description :sparkles:
[HL-1269](https://helsinkisolutionoffice.atlassian.net/browse/HL-1269?atlOrigin=eyJpIjoiYzAzNTc2M2ZkNDUwNDI1NzhiOTVmMWVhNTEzZWM5ZGEiLCJwIjoiaiJ9) 
Trying to update records in AHJO after new attachments were uploaded in Helsinkilisä failed, because the attachments uploaded after opening a case do not have a `ahjo_version_series_id `property. The `ahjo_version_series_id` is  needed to match the old records with the ones they are replaced with. 
This PR changes the update records request so that only the pdf_summary of the application is re-created and re-sent in the update records request (PUT request to AHJO) and all attachments uploaded after opening a case are sent as new records (POST request to AHJO). The new attachments are identified by the new `downloaded_by_ahjo`-attribute which is a datetime inserted after AHJO has downloaded an attachment.
The AHJO POST request for the new attachments can be scheduled to run every 15 minutes.

[HL-1271](https://helsinkisolutionoffice.atlassian.net/browse/HL-1271?atlOrigin=eyJpIjoiYWNlYmFlYzYwMzRiNGRkOThlMmEyNDFkNDAxMGNmZDkiLCJwIjoiaiJ9)
The Title-attribute of a record in the JSON-payload that is sent to AHJO defines what title is shown in the UI of AHJO.
This PR changes the formatting of the Application record and attachment record as requested by the people working with AHJO.

## Additional notes :spiral_notepad:

[HL-1269]: https://helsinkisolutionoffice.atlassian.net/browse/HL-1269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[HL-1271]: https://helsinkisolutionoffice.atlassian.net/browse/HL-1271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2935.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2935.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2935.dev.hel.ninja :laughing::tada::tada::tada:

File: backend/benefit/applications/api/v1/ahjo_integration_views.py [hunk 1]
```diff
@@ -150,10 +153,14 @@ def handle_success_callback(
             elif request_type == AhjoRequestType.SEND_DECISION_PROPOSAL:
                 ahjo_status = AhjoStatusEnum.DECISION_PROPOSAL_ACCEPTED
                 info = "Decision proposal was sent to Ahjo"
-            elif request_type == AhjoRequestType.UPDATE_APPLICATION:
-                self._handle_update_records_success(application, callback_data)
+            elif request_type in AhjoRequestType.UPDATE_APPLICATION:
```
Comment (sirtawast) [inline backend/benefit/applications/api/v1/ahjo_integration_views.py:156]: Why is this `in` while the others are `==`?

File: backend/benefit/applications/services/ahjo_integration.py [hunk 2]
```diff
@@ -511,25 +520,28 @@ def send_request_to_ahjo(
     timeout = settings.AHJO_REQUEST_TIMEOUT
     headers["Content-Type"] = "application/json"
 
+    data = json.dumps(data, default=str)
```
Comment (sirtawast) [inline backend/benefit/applications/services/ahjo_integration.py:523]: Good refactor!

**Review by sirtawast — APPROVED** [review]

File: backend/benefit/applications/api/v1/ahjo_integration_views.py (hunk 1 above)
Comment (rikuke) [reply backend/benefit/applications/api/v1/ahjo_integration_views.py:156]: Ahh I think I initially compared it with a list of AhjoRequestTypes, and left it like that.

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2935.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2935.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2935.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2935.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**